editor/components/block-list/breadcrumb.js

```diff
@@ -53,7 +52,7 @@ export class BlockBreadcrumb extends Component {
 	}
 
 	render( ) {
-		const { uid, rootUID, selectRootBlock, isHidden } = this.props;
```

Comment:
If we're not using `selectRootBlock`, the composed `withDispatch` can be removed.

Reply:
>If we're not using `selectRootBlock`, the composed `withDispatch` can be removed.

Removed in 11d7161d16b97b3b0486385bbc46c123586f889d.